### PR TITLE
Add basic Dashboard widget

### PR DESCRIPTION
**Pull request by eneiluj**
Hey. Here is a very basic an incomplete Dashboard widget for Social.

I only managed to produce 'Follow' type notifications on my test instance. So we need to make the component methods manage  other notification types.

social2

### CODE REVIEW

**Comment by daita** [comment]
I have some error during building:

$ eslint --ext .js,.vue src --fix

/home/maxence/PhpstormProjects/social/src/dashboard.js
  15:44  error  "@nextcloud/l10n" is extraneous   node/no-extraneous-import
  16:23  error  "./views/Dashboard" is not found  node/no-missing-import

/home/maxence/PhpstormProjects/social/src/views/Dashboard.vue
  26:21  error  'showSuccess' is defined but never used  no-unused-vars
  26:40  error  "@nextcloud/dialogs" is extraneous       node/no-extraneous-import
  27:8   error  'moment' is defined but never used       no-unused-vars

✖ 5 problems (5 errors, 0 warnings)

**Comment by eneiluj** [comment]
I'll fix that.

I don't know why eslint does not compile Social the same way in my context and yours. I tried from scratch and it still runs without errors:
```
git clone git@github.com:nextcloud/social -b enh/dashboard-widget --single-branch
cd social
make dev-setup
make build-js
```

Could it be there is some config that is only present locally on your side?

**Comment by eneiluj** [comment]
Lint is now successful.

File: img/add_user.svg [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 18.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+
+<svg
```
Comment (juliushaertl) [inline img/add_user.svg:4]: Could be minified :)

Comment (eneiluj) [reply img/add_user.svg]: Indeed :grin: Done

**Review by jancborchardt — COMMENTED** [review]
Nice! Looks great design-wise. Agree that the component should also be able to handle the other notification types too, like @-mention, direct message and such. :)

**Comment by jancborchardt** [comment]
@daita @icewind1991 can you do another review? :)

**Pull request merged**